### ISSUE TITLE
Migration to Scrapy

### ISSUE THREAD

**Issue by sagaragarwal94** [scrapeInfo/conference-scrape#3]
@schedutron  We need to migrate to scrapy and create spiders for the conference scrape and hacakthons scrape. What say ?

**Comment by schedutron** [comment]
Good idea. That would also promote more object-oriented design. I'll start working on it.

**Comment by sagaragarwal94** [comment]
@schedutron Yes sure. Let me push the introductory code today or by tomorrow morning to the repository and then you trigger the rest of it.

**Comment by schedutron** [comment]
@sagaragarwal94 Hey I've created a separate branch for migration in my fork of this repo. You can find the spider in the [conference_scraping/conference_scraping/spiders](https://github.com/schedutron/conference-scrape/tree/scrapy-migrate/conference_scraping/conference_scraping/spiders) directory.

In the migration branch, I've also updated the requirements file to include Scrapy and the README to include instructions for scraping with Scrapy.

Have a look and provide feedback. 🙂
